ksa_compliance/standard_doctypes/sales_invoice.py

```diff
@@ -106,17 +106,20 @@ def should_enable_zatca_for_invoice(invoice_id: str) -> bool:
 def calculate_tax_amount(self, method):
     if self.items:
         for item in self.items:
-            item_tax_template = frappe.get_doc("Item Tax Template", item.item_tax_template)
-            if len(item_tax_template.taxes) > 1:
-                frappe.throw(
-                    msg="""
-                        One or more items have a tax template with multiple tax accounts which is not currently 
-                        supported. Please contact the vendor.
-                        """,
-                    title="Multiple tax accounts found"
-                )
-            if item_tax_template.disabled:
-                frappe.throw("One or more items has disabled tax template", title="Disabled tax template")
-            item_tax_rate = item_tax_template.taxes[0].tax_rate
+            if not item.item_tax_template:
+                item_tax_rate = sum(account.rate for account in self.taxes)
+            else:
+                item_tax_template = frappe.get_doc("Item Tax Template", item.item_tax_template)
```

Comment:
We can make this more readable by extracting a method that accepts the tax template ID and returns the rate. That would make both branches of the if/else similar (`item_tax_rate = … `)